src/components/SignupEmailCard.tsx

```diff
@@ -0,0 +1,17 @@
+type UserData = {
+  email: string;
+};
+
+type EmailFormProps = {
+  user: UserData;
+  updateFields: (fields: Partial<UserData>) => void;
+};
+
+export const SignupEmailCard = ({ user, updateFields }: EmailFormProps) => {
+  return (
+    <div>
+      <label htmlFor="email">Email:</label>
+      <input id="email" required type="text" value={user.email} onChange={(e) => updateFields({ email: e.target.value })} />
```

Comment:
Does the code logic prefer (required type="text") or would it be better to  have (required type="email")
```suggestion
      <input id="email" required type="email" value={user.email} onChange={(e) => updateFields({ email: e.target.value })} />
```